packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -516,55 +519,57 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
         });
         commandRegistry.registerCommand(CommonCommands.CLOSE_TAB, {
             isEnabled: (event?: Event) => {
-                const tabBar = this.findTabBar(event);
+                const tabBar = this.contextMenuService.findTabBar(event);
```

Comment:
please revert changes to all unaffected commands

Reply:
Sorry for replying so late @akosyakov, the reason why I changed this was because, findTabBar and FindTitle needs to be used in the `navigator-contribution.ts` so in order to get rid of copy/pasting the methods, I tried to put it in a new file, just so it could be used by both the `core` and `navigator`. 
Would it be a better idea to put these methods to `application-shell.ts` just so it could be used in both the packages?